cspell.json

```diff
@@ -1,12 +1,59 @@
 {
   "version": "0.2",
   "language": "en",
-  "ignoreWords": [],
+  "ignoreWords": [
+    "psender",
+    "crytic",
+    "ABDK",
+    "IERC",
+    "ITWAP",
+    "IUAR",
+    "tippyjs",
+    "ALUSD",
+    "jsbi",
+    "JSBI",
+    "SjymJ5maJ4",
+    "iefix",
+    "Csvg",
+    "Cpath",
+    "proto",
+    "adoc",
+    "mixedcase",
+    "TYPEHASH",
+    "chainid",
+    "accu",
+    "implementat",
+    "devfund",
+    "setu",
+    "metapools",
+    "Mocku",
+    "reedem",
+    "retdata",
+    "ntype",
+    "zapaz",
+    "Qmanager",
+    "IUBQ",
+    "ADMGR",
+    "coef",
+    "Vtoken",
+    "transfered",
+    "Ronding",
+    "lprewards",
+    "structs",
+    "tlsv"
+  ],
   "ignorePaths": [
     "**/*.json",
     "**/*.log",
     "**/*.svg",
     "**/*.lock",
+    "**/packages/contracts/lib/**",
+    "**/packages/contracts/dollar/lib/**",
+    "**/packages/contracts/src/dollar/libs/**",
+    "**/packages/contracts/src/dollar/interfaces/**",
+    "**/packages/dapp/dist/**",
+    "**/packages/contracts/src/dollar/BondingFormulas.sol",
+    "**/cspell.json",
```

Comment:
I don't understand your plan with skipping the contracts, but the contracts should be included in this bounty.

Reply:
The plan for skipping contracts for now is that changing contract properties unless @0xcodercrane clarifies that he wishes to include them. The language used in comments has not been declarative.

`packages/contracts/lib`, `packages/contracts/dollar/lib`, and `packages/contracts/src/dollar/libs` appear to be a library, which i thought was requested to not be updated. Am i misidentifying these folders perhaps?

`BondingFormulas.sol` was cruft that should have been removed, and is now, if you are wondering about that file

Reply:
packages/contracts/lib is same to node_modules so we can ignore it. but packages/contracts/src/dollar/libs needs to be included there imo.

Reply:
hmm I think you can add cspell ignore for `packages/contracts/src/dollar/libs` files because its moved to `packages/contracts/lib` in https://github.com/ubiquity/ubiquity-dollar/pull/302 pr

Reply:
i happened to push these inclusions just a few minutes ago. just force pushed to revert and match your comments above.